src/common.cpp

```diff
@@ -21,15 +21,19 @@
 //    GNU General Public License for more details.                          //
 /****************************************************************************/
 
+#define _WIN32_WINNT 0x0A00
```

Comment:
Do we need to define it? If yes, why?

Reply:
And 0x0A00 seems pretty high, it means we support only Windows 10+

Reply:
I added it for this macro SM_CXPADDEDBORDER (https://github.com/julianxhokaxhiu/FFNx/blob/cf2a40f75882e5a4af926f891ce8e0a3ad0c15ee/src/common.cpp#L479). If you go into winuser.h you will find that it is behind and `ifndef` (https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-getsystemmetrics). I did put a quite high value, but that is taken from renderer.cpp (https://github.com/julianxhokaxhiu/FFNx/blob/cf2a40f75882e5a4af926f891ce8e0a3ad0c15ee/src/renderer.cpp#L29). But maybe the one in renderer.cpp is not an issue.

I think it became necessary because I moved "renderer.h" at first include and something inside there is defining the _WIN32_WINNT to a low value. 

I will put the minimum value required for the macro.

Reply:
Actually I will try to find a way to avoid putting this macro so that it will be the same.

Reply:
I removed that macro and also put renderer.h below as I have changed the include inside the renderer.h file. It makes more sense to import common.h first since it contains common stuff.

Reply:
Thanks, I think the define would have had no impact anyway but better not have it around hardcoded. Appreciated!